radix-engine/src/system/system_modules/costing/fee_table.rs

```diff
@@ -220,88 +199,139 @@ impl FeeTable {
     }
 
     #[inline]
-    pub fn create_node_cost(
-        &self,
-        _node_id: &NodeId,
-        total_substate_size: usize,
-        store_access: &StoreAccessInfo,
-    ) -> u32 {
-        add3(
-            500,
-            Self::data_processing_cost(total_substate_size),
-            Self::store_access_cost(store_access),
-        )
+    pub fn create_node_cost(&self, event: &CreateNodeEvent) -> u32 {
+        match event {
+            CreateNodeEvent::Start(_node_id, node_substates) => {
+                let total_substate_size = node_substates
+                    .values()
+                    .map(|x| x.values().map(|x| x.len()).sum::<usize>())
+                    .sum::<usize>();
```

Comment:
Not necessarily related to your PR, but we need to double check if we've counted the size of child nodes.